### DISCUSSION TITLE
TypeError: index.getObject is not a function

### DISCUSSION THREAD

**Discussion by miag676** [algolia/instantsearch#5288] [Q&A] [answered]
I'm using algolia in my react application and would like to get object by id, but I get the following error: 
`TypeError:index.getObject is not a function`.
This is the part of my code related to the issue:
```
import algoliasearch from 'algoliasearch/lite';
const searchClient = algoliasearch(
  '***',
  '***'
);
const index = searchClient.initIndex('Besedilo')

export default function Product() {

  useEffect(() => {
        const path = window.location.pathname.replace('/product/', '')
        index.getObject(path).then(object => {
          console.log(object);
        });
        setLoading(false);
  });
}
```

Can anyone help me resolve the error?
Thanks in advance!

**Comment by Haroenv** [comment, marked as answer]
getObject isn't a function of algoliasearch/lite, only of the "full" version of algoliasearch. What you can do instead is searching with the objectID as a filter ``search({ filters: `objectID:${path}`})``, that way you can still use the lite version of algoliasearch.

**Reply by miag676** [reply]
This solved the problem. Thank you!
